frame/support/src/storage/mod.rs

```diff
@@ -30,6 +30,83 @@ pub mod child;
 pub mod generator;
 pub mod migration;
 
+#[cfg(not(build_type = "release"))]
```

Comment:
This feature doesn't exist in Rust.

You want to use `debug_assertions`

Reply:
I was referring to https://github.com/paritytech/substrate/blob/87063c3c00da34213379330bae3174aa0da7ad0f/client/consensus/slots/src/lib.rs#L272

Will change to `debug_assertions` instead